stripe-patrons-data/src/main/scala/com/gu/patrons/lambdas/PatronSignUpEventLambda.scala

```diff
@@ -40,26 +40,26 @@ import scala.concurrent.duration.{Duration, DurationInt, MINUTES}
 import scala.jdk.CollectionConverters.MapHasAsScala
 import scala.util.{Failure, Success, Try}
 
-object PatronSignUpEventLambda extends StrictLogging {
+class PatronSignUpEventLambda extends StrictLogging {
   val runner = configurableFutureRunner(60.seconds)
 
+  implicit val stage = StageConstructors.fromEnvironment
+  private val stripeConfig = PatronsStripeConfig.fromParameterStoreSync(stage)
+  private val identityConfig = PatronsIdentityConfig.fromParameterStoreSync(stage)
```

Comment:
I am wondering in my mind how easy it would be to write an automated test for this.  It would be very easy to regress this if it were implemented in a critical lambda and I'm wondering how we would know before we deploy - it could get expensive quickly plus slow for users.

Reply:
I see what you mean. The difference between code that gets executed during initialisation or not is quite subtle.
But we don't want the tests to be talking to AWS. I will keep thinking about this one...

Reply:
It's not as good as a test, but we could probably create cloudwatch alarms that are triggered when a lambda's duration exceeds some threshold, which would suggest something has stopped happening at initialisation time